content/docs/guides/guacamole.md

```diff
@@ -0,0 +1,164 @@
+   ```bash
+   git clone https://github.com/boschkundendienst/guacamole-docker-compose
+   ```
+1. Go into your project and run the provided `prepare.sh` script. This script initializes a PostgreSQL database that works with Guacamole. (It generates also a self-signed certificate and private key for Nginx, which you can ignore.)
+
+   ```bash
+   cd guacamole-docker-compose
+   ./prepare.sh
+   ```
+
+1. In the `guacamole` container, add the following environment variables:
+
+```yaml title="docker-compose.yaml"
+guacamole:
+  container_name: guacamole_compose
+  depends_on:
+    - guacd
+    - postgres
+  environment:
+    GUACD_HOSTNAME: guacd
+    POSTGRES_DATABASE: guacamole_db
+    POSTGRES_HOSTNAME: postgres
+    POSTGRES_PASSWORD: 'ChooseYourOwnPasswordHere1234'
+    POSTGRES_USER: guacamole_user
+    # highlight-start
+    # Enables HTTP header authentication
+    HEADER_ENABLED: true
+    # Adds expected HTTP header from incoming Pomerium requests
+    HTTP_AUTH_HEADER: X-Pomerium-Claim-Email
+    # highlight-end
+  image: guacamole/guacamole
+  networks:
+    - guacnetwork_compose
+  volumes:
+    - ./record:/record:rw
+  ports:
+    - 8080/tcp
+  restart: always
+```
+
+## Configure Pomerium
+
+In your Docker Compose file, replace `nginx` with Pomerium Core:
+
+1. Remove the `nginx` configuration:
+   ```yaml
+   # nginx
+   nginx:
+     container_name: nginx_guacamole_compose
+     restart: always
+     image: nginx
+     volumes:
+       - ./nginx/templates:/etc/nginx/templates:ro
+       - ./nginx/ssl/self.cert:/etc/nginx/ssl/self.cert:ro
+       - ./nginx/ssl/self-ssl.key:/etc/nginx/ssl/self-ssl.key:ro
+     ports:
+       - 8443:443
+     links:
+       - guacamole
+     networks: guacnetwork_compose
+   ```
+1. Add the Pomerium Core configuration:
+
+   ```yaml showLineNumbers
+   pomerium:
+     image: cr.pomerium.com/pomerium/pomerium:latest
+     # highlight-start
+     # Mount your config file
+     volumes:
+       - ./config.yaml:/pomerium/config.yaml:ro
+     # highlight-end
+     ports:
+       - 443:443
+     networks:
+       - guacnetwork_compose
+     environment:
+       JWT_CLAIMS_HEADERS: email
+   ```
+
+1. In line 5 above, you mount a Pomerium configuration file into Docker Compose. In your project's root directory, create a `config.yaml` file with the following configuration:
+
+   ```yaml title="Pomerium configuration file"
+   authenticate_service_url: https://authenticate.pomerium.app
+
+   jwt_claim_headers: email
+
+   routes:
+     - from: https://guacamole.localhost.pomerium.io
+       to: http://guacamole:8080
+       policy:
+         - allow:
+             or:
+               - email:
+                   # highlight-next-line
+                   is: user@example.com
+       pass_identity_headers: true
+   ```
+
+   Don't forget to replace `user@example.com` with your own email address.
+
```

Comment:
Can you show the full working configurations / yamls at the end?

Reply:
Yes, I'll add a section to do that.